workers/check_for_new_commits_on_github_worker.rb

```diff
@@ -1,6 +1,7 @@
 require_relative "worker_base"
 require_relative "../services/build_service"
 require_relative "../shared/models/provider_credential"
+require_relative "../shared/models/job_trigger"
```

Comment:
I noticed the file's name is `job_trigger`, but the class name is `CommitJobTrigger`. Was there a reason for that, or could we change that?